apps/generator/Dockerfile

```diff
@@ -8,14 +6,17 @@ WORKDIR /app
 ENV PUPPETEER_EXECUTABLE_PATH /usr/bin/chromium-browser
 ENV PUPPETEER_SKIP_CHROMIUM_DOWNLOAD true
 
+# Include the ARG instruction in this build stage
```

Comment:
please include one line comment explaining why it should be included in build stage